packages/react-components/react-tags/library/src/components/InteractionTag/InteractionTag.types.ts

```diff
@@ -28,6 +28,13 @@ export type InteractionTagProps<Value = TagValue> = ComponentProps<Partial<Inter
    */
   disabled?: boolean;
 
+  /**
+   * An InteractionTag can be selected.
+   *
+   * @default false
+   */
+  selected?: boolean;
```

Comment:
Based on Amber's feedback:

```suggestion
  /**
   * An InteractionTag can be selected.
   * Note: This prop only changes the appearance of the tag at the moment. A future PR will add the integration with TagGroup.
   *
   * @default false
   */
  selected?: boolean;
```